src/pages/promiseDetail/PromiseDetailPageJunior.tsx

```diff
@@ -14,28 +14,22 @@ import Loading from '@components/commons/Loading';
 import { useGetGoogleMeetLink } from '@pages/promiseList/hooks/queries';
 
 const PromiseDetailPageJunior = () => {
-  // 데모데이
-  const googleMeetLink = 'https://meet.google.com/gnr-wgcr-gcv';
-  const onClickLink = () => {
-    window.open(googleMeetLink, '_blank');
-  };
-
   // 라우터 이동할 때 location으로 약속id, 눌린 탭 상태값(pending, sheduled, ..) 받아와야함
   const navigate = useNavigate();
   const location = useLocation();
   const tap = location.state.tap;
   const myNickname = location.state.myNickname;
   const appointmentId = location.state.appointmentId;
   const seniorId = location.state.seniorId;
```

Comment:
```suggestion
const { tap, myNickname, appointmentId, seniorId } = location.state || {};
```
P4) location.state는 브라우저 새로고침 시 데이터가 사라진다고 해요! 이런 경우에 location.state는 undefined가 될 수 있기 때문에 코드에서 location.state가 undefined일 수 있다는 점을 고려해주면 오류 처리에 좋을거 같아요!

Reply:
useNavigate로 state를 전달하면 새로고침시에 state가 유실되지 않아요! 그 history 객체를 기반으로 동작하기 때문에 새로고침이 아니라 뒤로가기나, url이 변경되는 등 popstate의 동작이 일어날 때만 state가 제거된답니다-!

Reply:
[https://developer.mozilla.org/ko/docs/Web/API/Window/popstate_event](https://developer.mozilla.org/ko/docs/Web/API/Window/popstate_event)

[https://developer.mozilla.org/ko/docs/Web/API/History/state](https://developer.mozilla.org/ko/docs/Web/API/History/state)

참고~!